### PR TITLE
Update the code with missed functionality

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
- Update the code with missed functionality
- Add config files required for ballerina central

### CODE REVIEW

File: soap/soap_connector.bal [hunk 1]
```diff
@@ -32,10 +32,27 @@ public type SoapConnector object {
         P{{request}} Request to be sent
         R{{}} If success, returns the response object, else returns `SoapError` object
     }
-    public function sendSoapRequest(string path, SoapRequest request) returns (SoapResponse|SoapError);
+    public function sendReceive(string path, SoapRequest request) returns (SoapResponse|SoapError);
```
Comment (riyafa) [inline soap/soap_connector.bal]: Brackets are used only for tuples. You can remove them

File: soap/soap_connector.bal [hunk 2]
```diff
@@ -48,3 +65,23 @@ public function SoapConnector::sendSoapRequest(string path, SoapRequest request)
         }
     }
 }
+
+public function SoapConnector::sendRobust(string path, SoapRequest request) returns (()|SoapError) {
```
Comment (riyafa) [inline soap/soap_connector.bal]: You can use `returns error?`

Comment (ldclakmal) [reply soap/soap_connector.bal]: Then I have to return an `error` definitely. But in the implementation of this method there is a scenario where `nill` should be returned. So, how can I resolve that?

File: soap/soap_connector.bal (hunk 1 above)
Comment (ldclakmal) [reply soap/soap_connector.bal]: Fixed.

File: soap/soap_connector.bal (hunk 2 above)
Comment (riyafa) [reply soap/soap_connector.bal]: `error?` means error or nil. It is a short form.

Comment (ldclakmal) [reply soap/soap_connector.bal]: Got it and fixed. Thanks.

File: soap/soap_connector.bal [hunk 3]
```diff
@@ -32,10 +32,26 @@ public type SoapConnector object {
...
         R{{}} If success, returns the response object, else returns `SoapError` object
     }
-    public function sendSoapRequest(string path, SoapRequest request) returns (SoapResponse|SoapError);
+    public function sendReceive(string path, SoapRequest request) returns SoapResponse|SoapError;
+
+    documentation {
+        Send Robust requests.Sends the request and possibly receives an error.
+        P{{path}} Resource path
+        P{{request}} Request to be sent
+        R{{}} If success, returns nothing, else returns `SoapError` object
```
Comment (Manuri) [inline soap/soap_connector.bal]: Can we say returns nil instead of "nothing"?

File: Ballerina.toml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+[project]
+license = "Apache-2.0"
+org-name = "wso2"
+version = "0.3.0"
+authors = ["WSO2"]
+homepage = "https://wso2.com/"
+repository = "https://github.com/wso2-ballerina/package-soap"
+keywords = ["soap"]
```
Comment (Shan1024) [inline Ballerina.toml]: I think it is better to add a newline here. Is there any limitation for that in toml files?

File: soap/soap_connector.bal (hunk 3 above)
Comment (ldclakmal) [reply soap/soap_connector.bal]: Fixed

File: Ballerina.toml (hunk 4 above)
Comment (ldclakmal) [reply Ballerina.toml]: Fixed

**Pull request merged**